app/serializers/spotlight/exhibit_export_serializer.rb

```diff
@@ -34,6 +34,10 @@ class ExhibitExportSerializer < Roar::Decorator
       property prop
     end
 
+    property :theme, if: lambda { |options|
```

Comment:
I think `if` applies for both serializing + deserializing. It looks like `setter` may be just for serializing? Something like:

`property :theme, setter: ->(fragment:, represented:, **) { represented.theme = fragment if Spotlight::Engine.config.exhibit_themes.include? fragment }`

Reply:
We need it for deserializing though, right?  The serialized export will include the theme since the source instance had it, so the target instance needs to be able to ignore it.

Reply:
Indeed -- I was thinking there's no need to check during serialization.

Reply:
Oh, so `setter` is for deserialization, and `getter` for serialization? Am I reading http://trailblazer.to/gems/representable/3.0/function-api.html#getter right?

Reply:
Yes, I believe that's correct.

Reply:
righto, updated